src/registrar/templates/includes/non-production-alert.html

```diff
@@ -0,0 +1,5 @@
+<div class="usa-alert usa-alert--warning usa-alert--slim margin-y-0 {% if add_class %}{{ add_class }}{% endif %}">
+    <div class="usa-alert__body {% if add_body_class %}{{ add_body_class }}{% endif %}">
+        You are not on production.
```

Comment:
```suggestion
        <b>WARNING:</b> You are not on production.
```
(Optional) I think this contrasts a bit better

Reply:
I like this. I'll commit and see what the designers think.